.github/workflows/ci.yml

```diff
@@ -652,7 +729,8 @@ jobs:
       - run: python -m pytest modin/experimental/spreadsheet/test/test_general.py
 
   upload-coverage:
-    needs: [test-internals, test-api-and-no-engine, test-defaults, test-hdk, test-all-unidist, test-all, test-experimental, test-cloud]
+    needs: [test-internals, test-api-and-no-engine, test-defaults, test-hdk, test-all-unidist, test-all, test-experimental, test-cloud, test-sanity]
+    if: always()  # we need to run it regardless of some job being skipped, like in PR
```

Comment:
Why?

Reply:
Because if I do not specify a condition, by default this job will get skipped if any of its `needs` are skipped, which is most likely happening in a PR (where `test-all-unidist` is skipped).